src/api/app/models/notification.rb

```diff
@@ -0,0 +1,55 @@
+class Notification < ApplicationRecord
+  belongs_to :user
+  belongs_to :group
+
+  def subscriber=(subscriber)
+    if subscriber.is_a? User
+      self.user = subscriber
+    elsif subscriber.is_a? Group
+      self.group = subscriber
+    end
+  end
+end
+
+class Notification::RssFeedItem < Notification
```

Comment:
I know I started this code with both classes in the same but since there is a significant amount of code in `Notification::RssFeedItem` can you please move it to its own file?